site/en/docs/handbook/how-to/add-an-author/index.md

```diff
@@ -3,35 +3,57 @@ layout: 'layouts/doc-post.njk'
 title: Add an author
 description: 'Add an author to a post.'
 date: 2020-12-07
+updated: 2021-09-08
 ---
 
-## Add an Author
-
-- Follow the [guide to uploading
-  media](/docs/handbook/how-to/add-media/#navigate-to-the-media-uploader) to upload your profile
-  photo to our image CDN.
-    - Images should be square and less than 5Mb.
-- After the image is uploaded, copy the src, it should look like this: `image/foR0vJZKULb5AGJExlazy1xYDgI2/1603484068246.jpg`.
-- Open `site/_data/authorsData.json` and add a new entry to the bottom of the file. You can copy this one as a starting point. If you don't have one of the social properties you can omit it.
-
-```json
-"robdodson": {
-  "name": {
-    "given": "Rob",
-    "family": "Dodson"
-  },
-  "org": {
-    "name": "Google",
-    "unit": "Developer Relations"
-  },
-  "descriptions": {
-    "en": "Describe yourself. This text will appear next to your photo on posts."
-  },
-  "country": "US",
-  "homepage": "https://robdodson.me",
-  "twitter": "rob_dodson",
-  "github": "robdodson",
-  "glitch": "robdodson",
-  "image": "image/foR0vJZKULb5AGJExlazy1xYDgI2/hdnOUruuCIijvnGWvleC.jpg"
-}
-```
\ No newline at end of file
+## Add yourself to the authors list
+
+1. Add a new object to [`authors.yml`](https://github.com/GoogleChrome/developer.chrome.com/blob/main/site/_data/i18n/authors.yml) with the following structure. Make sure to choose a unique author slug.
+
+   ```yml
+   authorslug:
+     title:
+       en: Full Name
+     description:
+       en: A relevant description about yourself you'd like to share.
+   ```
+
+   You can add multiple titles and descriptions to your author data for each language that is supported by using the [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) code for that language.
+
+   ```yml
+   authorslug:
+     title:
+       en: Full Name
+       es: Nombre completo
+     description:
+       en: A relevant description about yourself you'd like to share.
+       es: Una descripción relevante traducida por Google sobre usted que le gustaría compartir.
+   ```
+
+2. If you want links to your online accounts to appear in your author lockup add a new object to [`authorsData.json`](https://github.com/GoogleChrome/developer.chrome.com/blob/main/site/_data/authorsData.json) with the following structure. Make sure to use the same author slug used in the [`authors.yml`](https://github.com/GoogleChrome/developer.chrome.com/blob/main/site/_data/i18n/authors.yml).
+
+   ```json
+   "jaimiesmith": {
+     "twitter": "jaimiesmith",
+     "github": "jaimiesmith",
+     "glitch": "jaimiesmith",
+     "homepage": "https://jaimiesmithis.cool/",
```

Comment:
nit: no trailing comma here (even though I'm sad about JSON not supporting this)